streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -99,8 +99,7 @@
                                 keySerde,
                                 valSerde,
                                 sourceName.equals(this.name) ? sourceNodes : Collections.singleton(sourceName),
-                                storeBuilder.name(),
-                                isQueryable,
+                                queryableStoreName,
```

Comment:
I think we should make sure (to avoid bugs), that `queryableStoreName == null || queryableStoreName.equals(storeBuilder.name()`) -- can we add a check? Not sure if this class is the best place for it? But maybe it's also good to have multiple checks in place?

Reply:
It's a matter of code style (I personally like assertions, but many people don't). Do we run test with assertions enabled? If not, it does not make sense to add one.